ssdutil/main.py

```diff
@@ -37,44 +77,52 @@ def import_ssd_api(diskdev):
         sys.path.append(os.path.abspath(platform_plugins_path))
         from ssd_util import SsdUtil
     except ImportError as e:
-        log.log_warning("Platform specific SsdUtil module not found. Falling down to the generic implementation")
+        log.log_warning("Platform specific SsdUtil module not found. Falling down to the generic implementation")  # noqa: E501
         try:
-            from sonic_platform_base.sonic_storage.ssd import SsdUtil
+            from sonic_platform_base.sonic_ssd.ssd_generic import SsdUtil
         except ImportError as e:
-            log.log_error("Failed to import default SsdUtil. Error: {}".format(str(e)), True)
-            raise e
+            try:
+                from sonic_platform_base.sonic_storage.ssd import SsdUtil
+            except ImportError as e:
+                log.log_error("Failed to import default SsdUtil. Error: {}".format(str(e)), True)  # noqa: E501
+                raise e
```

Comment:
@assrinivasan why we need two different SsdUtil implementation?

Reply:
I was going to say to maintain backwards compatibility, but since you said we should only take this into master and 202405+, there is no need for SsdUtil anymore. I will therefore change this to just import StorageUtil.